xla/python/py_array.cc

```diff
@@ -701,6 +704,99 @@ py::dict PyArray::CudaArrayInterface() {
   return result;
 }
 
+StatusOr<pybind11::object> CudaArrayInterfaceToBuffer(const pybind11::dict& cai, std::shared_ptr<PyClient> client) {
+#ifndef GOOGLE_CUDA
+    throw XlaRuntimeError("This operation requires CUDA support.");
+#else
+  if (!cai.contains("data")) {
+    return InvalidArgument("CUDA Array Interface does not define `data`");
```

Comment:
Could you please use `absl::InvalidArgumentError` instead? We are trying to standardizing the calls. See below for examples
https://github.com/search?q=repo%3Aopenxla%2Fxla%20%22absl%3A%3Ainvalidargumenterror%22&type=code

Same for other places.

Reply:
Sure, thanks for the hint!
Fixed.